test/unit/uk/gov/hmrc/apinotificationqueue/controller/QueueControllerSpec.scala

```diff
@@ -18,22 +18,33 @@ package uk.gov.hmrc.apinotificationqueue.controller
 
 import play.api.http.Status
 import play.api.test.FakeRequest
-import play.api.http.Status
-import play.api.test.FakeRequest
-import uk.gov.hmrc.play.test.UnitSpec
-import uk.gov.hmrc.play.test.WithFakeApplication
 import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
-class MicroserviceHelloWorldControllerSpec extends UnitSpec with WithFakeApplication{
+class QueueControllerSpec extends UnitSpec with WithFakeApplication {
 
-  val fakeRequest = FakeRequest("GET", "/")
 
-  "GET /" should {
+  "GET /messages" should {
```

Comment:
In my opinion this should return 200 also in the case of an empty list

Reply:
We don't currently have this functionality when we do we should you are correct.